assets/prometheus-user-workload/cluster-role.yaml

```diff
@@ -52,6 +52,14 @@ rules:
   - alertmanagers
   verbs:
   - get
+- apiGroups:
```

Comment:
instead of granting cluster-wide permissions, it would be better for CMO to bind the `monitoring-alertmanager-edit` role to the prometheus-user-workload service account.

Reply:
but that is a role which is bound to the openshift-monitoring namespace. it is not a cluster role so i dont think it can be bound outside its namespace.

Reply:
Unless I'm mistaken (https://kubernetes.io/docs/reference/access-authn-authz/rbac/#kubectl-create-rolebinding), it should be possible to bind a role from the openshift-monitoring namespace to a service account in the openshift-user-workload-monitoring namespace.

```
apiVersion: rbac.authorization.k8s.io/v1
kind: RoleBinding
metadata:
  name: prometheus-user-workload-alertmanager-edit
  namespace: openshift-monitoring
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: Role
  name: monitoring-alertmanager-edit
subjects:
- kind: ServiceAccount
  name: prometheus-user-workload
  namespace: openshift-user-workload-monitoring

```